src/app/admin/packingSlotsTable/PackingSlotsTable.tsx

```diff
@@ -326,26 +337,60 @@ const PackingSlotsTable: React.FC = () => {
         <>
             {errorMessage && <ErrorSecondaryText>{errorMessage}</ErrorSecondaryText>}
             {rows && (
-                <DataGrid
-                    rows={rows}
-                    columns={packingSlotsColumns}
-                    editMode="row"
-                    rowModesModel={rowModesModel}
-                    onRowModesModelChange={setRowModesModel}
-                    onRowEditStop={handleRowEditStop}
-                    processRowUpdate={processRowUpdate}
-                    slots={{
-                        toolbar: EditToolbar,
-                        loadingOverlay: LinearProgress,
-                    }}
-                    slotProps={{
-                        toolbar: { setRows, setRowModesModel, rows },
-                    }}
-                    loading={isLoading}
-                />
+                <DataGridStyling>
+                    <DataGrid
+                        rows={rows}
+                        columns={packingSlotsColumns}
+                        editMode="row"
+                        rowModesModel={rowModesModel}
+                        onRowModesModelChange={setRowModesModel}
+                        onRowEditStop={handleRowEditStop}
+                        processRowUpdate={processRowUpdate}
+                        slots={{
+                            toolbar: EditToolbar,
+                            loadingOverlay: LinearProgress,
+                        }}
+                        slotProps={{
+                            toolbar: { setRows, setRowModesModel, rows },
+                        }}
+                        loading={isLoading}
+                        getRowClassName={(params) =>
+                            (params.indexRelativeToCurrentPage + 1) % 2 === 0 ? "even" : "odd"
+                        }
+                        hideFooter
+                    />
+                </DataGridStyling>
             )}
         </>
     );
 };
 
+const DataGridStyling = styled.div`
```

Comment:
Can we do `DataGrid.div` and use that instead, rather than creating a wrapper element around the data grid?

Reply:
just realised my suggestion was way off

Reply:
sorry